diff --git a/node_modules/conventional-changelog-angular/writer-opts.js b/node_modules/conventional-changelog-angular/writer-opts.js
index d5de1a3..7c52896 100644
--- a/node_modules/conventional-changelog-angular/writer-opts.js
+++ b/node_modules/conventional-changelog-angular/writer-opts.js
@@ -4,6 +4,8 @@ const compareFunc = require('compare-func')
 const Q = require('q')
 const readFile = Q.denodeify(require('fs').readFile)
 const resolve = require('path').resolve
+// I hate this. But transforms cannot be async... :/
+const {default: request} = require('sync-request');
 
 module.exports = Q.all([
   readFile(resolve(__dirname, './templates/template.hbs'), 'utf-8'),
@@ -69,13 +71,31 @@ function getWriterOpts () {
         let url = context.repository
           ? `${context.host}/${context.owner}/${context.repository}`
           : context.repoUrl
+
+        let [owner, repo] = context.repository ? [context.owner, context.repository] : context.repoUrl.split('/').slice(3);
+
+        let currentIssue;
         if (url) {
           url = `${url}/issues/`
+
           // Issue URLs.
           commit.subject = commit.subject.replace(/#([0-9]+)/g, (_, issue) => {
-            issues.push(issue)
+            issues.push(issue);
+            currentIssue = issue;
             return `[#${issue}](${url}${issue})`
           })
+
+          try {
+            const apiResult = request('GET', `https://api.github.com/repos/${owner}/${repo}/issues/${currentIssue}`, {
+              headers: {
+                'User-Agent': 'octokit-rest.js/18.12.0 octokit-core.js/3.5.1 Node.js/16.13.2 (win32; x64'
+              }
+            });
+
+            const data = JSON.parse(apiResult.getBody('utf8'));
+
+            commit.subject += ` by @${data.user.login}`;
+          } catch {}
         }
         if (context.host) {
           // User URLs.
