diff --git a/node_modules/docusaurus-plugin-typedoc-api/lib/components/MemberDeclaration.js b/node_modules/docusaurus-plugin-typedoc-api/lib/components/MemberDeclaration.js
index c42d124..ba78fb2 100644
--- a/node_modules/docusaurus-plugin-typedoc-api/lib/components/MemberDeclaration.js
+++ b/node_modules/docusaurus-plugin-typedoc-api/lib/components/MemberDeclaration.js
@@ -38,6 +38,15 @@ function MemberDeclaration({
   const minimal = useMinimalLayout.useMinimalLayout();
   const showTypes = reflection.typeParameter && reflection.typeParameter.length > 0;
   const showDeclaration = !minimal && ((_reflection$type = reflection.type) === null || _reflection$type === void 0 ? void 0 : _reflection$type.declaration);
+
+  if (!reflection.type) {
+    // TEMPORARY: define type as literal
+    reflection.type = {
+      type: 'literal',
+      value: typeof reflection.defaultValue !== 'undefined' ? reflection.defaultValue : 'unknown type',
+    }
+  }
+
   return /*#__PURE__*/jsxRuntime.jsxs(jsxRuntime.Fragment, {
     children: [/*#__PURE__*/jsxRuntime.jsx("div", {
       className: "tsd-panel-content",
